packages/amplify-e2e-tests/src/init/amplifyPush.ts

```diff
@@ -1,7 +1,7 @@
 import * as nexpect from 'nexpect';
 import { getCLIPath, isCI } from '../utils';
 
-export default function amplifyPush(
+function amplifyPush(
```

Comment:
We could have a single function for push and use `--yes` flag, which prevents it from asking any questions

Reply:
Agreed, but I think this can be added later.

Reply:
👍